spec/predicates/human-review-vcs.md

```diff
@@ -0,0 +1,155 @@
+# Predicate type: VCS Code Reviews
+
+Type URI: (tentative) https://in-toto.io/attestation/human-review/vcs/v0.1
+
+Version: 0.1.0
+
+## Purpose
```

Comment:
Given the Crev dependency review predicate is available as well, I think it would be helpful to add a short comment here about the complementary relationship between that and this predicate.